perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/build.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+sudo apt-get update
+
+echo "Installing git"
+sudo apt-get install git
+
```

Comment:
Why did we remove the go, fio installation?

Reply:
Because it shifted to related scripts in current implementation. https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/run_load_test_and_fetch_metrics.sh#L20